src/components/SearchBox/StyledSearchBox.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+const StyledSearchBox = styled.form`
+display: flex;
+justify-content: flex-end;
+align-items: flex-end;
+flex-direction: column;
+padding: 15px 30px;
+border: 1px solid black;
```

Comment:
Can you add in the use of the theme provider? 
For example, for the color the border is using.

This is from the first theming example from here: [https://styled-components.com/docs/advanced#theming](url)

` /* Color the border and text with theme.main */`

 ` color: ${props => props.theme.main};`
  `border: 2px solid ${props => props.theme.main};`

Reply:
I'd like to wait until theme is a bit more set up if you're not against that.
100% on board to using theme.js as heavily as possible but since I haven't built that out yet, I'd like to hold off for now